frontend/src/metabase/common/components/performance/SidebarCacheSection.tsx

```diff
@@ -0,0 +1,55 @@
+import type { Dispatch, SetStateAction } from "react";
```

Comment:
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/utGnP2ljFMMNOgi82vVn/aa14c2fa-4b14-44e6-9df1-6506908e9bb7.png)

This is the form-launching component found in both the dashboard and question sidebars